app/main/services/UpdateService.js

```diff
@@ -152,6 +152,7 @@ export default class UpdateService extends ServiceBase {
                     console.log('err response', response);
                     const errMsg = response && response.message ? response.message : (response && response.Error ? response.Error : '');
                     console.info(`Failure checking for updates: ${errMsg}`);
+                    log.error(`Failure checking for updates: ${errMsg}`);
```

Comment:
log.error shouldn't be used for this (), since IDE can be run from internal networks with no access to internet. Therefore such error should not be displayed to the user.
log.warn is more appropriate because it doesn't display error dialog in production mode